homeassistant/components/panasonic_viera/media_player.py

```diff
@@ -101,7 +101,7 @@ async def async_update(self):
 
     async def async_turn_on(self):
         """Turn on the media player."""
-        await self._remote.async_turn_on()
+        await self._remote.async_turn_on(self._context)
```

Comment:
```suggestion
        await self._remote.async_turn_on(context=self._context)
```

Reply:
So this is an instance of their own class which I have updated to take the context.

Reply:
Still applied it because it is also named context so works and makes it clearer what's going on :)

Reply:
Ah, ok, didn't realize that. Just looked like an oversight given all the other changes like it.